src/screens/Movie/Movie.tsx

```diff
@@ -155,10 +155,10 @@ const Movie = (): JSX.Element => {
         {playlist ? (
           <>
             <div className={styles.related}>
-              <h3>{playlist.title}</h3>
+              <h3>{isPlaylistPlaceholderData || playlistLoading ? t('common:loading') : playlist.title}</h3>
```

Comment:
The 'Loading' here feels a little weird to me, since it's on a header on a secondary part of the page. I think the suggestion below achieves the same thing without troubling the user with status of something extra. Ideally we would have a grey flashy empty text placeholder block for this and for the playlist item titles, but in the meantime, I think this is an easy fix.

```suggestion
              <h3>{playlist.title || '\u00A0'}</h3>
```

Reply:
Yes, good suggestion